public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -46,6 +50,7 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
   constructor(private route: ActivatedRoute, private api: DataBrowserService,
     private tooltipText: TooltipService,
     public dbc: DbConfigService) {
+    gtag('config', environment.gaId, 'auto');
```

Comment:
Could you check whether you can remove this? This seems like a global setting being applied in this very specific page component.

Reply:
Yes, it works when i remove this. Thanks.